utils/identity.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+)
+
+// GenerateIdentity creates identity file using radicle-keyutil
+// and saves it under keys/org.key
+func GenerateIdentity(org string) {
+	path := fmt.Sprintf("./keys/%s.key", org)
+	if _, err := os.Stat(path); err == nil {
+		// it exists, don't overwrite the key
+		return
+	}
+	cmd := exec.Command("radicle-keyutil", "--filename", path)
```

Comment:
The org node generates this automatically (I need to udpate the README). I think it would be better if we used that mechanism, as the `radicle-bins` repo is unmaintained. If we really want to have a utility to generate the keys locally to then send them over securily, maybe we port this code to a repo we are maintaining..

Reply:
Explained [above ^](https://github.com/radicle-dev/radicle-cloud/pull/1#discussion_r756074020)